### ISSUE TITLE
Cannot post UTF-8 chars in comments to notes and changesets

### ISSUE THREAD

**Issue by Zverik** [openstreetmap/openstreetmap-website#3184]
See https://github.com/openstreetmap/operations/issues/524

> For example, If I go to https://www.openstreetmap.org/note/2639402 and try to comment with "test with ccaron č letter" I get 401 Unauthorized. Same if it contains other national letters like "ČĆŠĐŽ". If I use just plain US-ASCII, it works fine.

**Comment by mmd-osm** [comment]
Issue is triggered as a side effect of https://github.com/openstreetmap/openstreetmap-website/commit/ad6c0d3ebabdfe985ce91b10e3ae07b68ebd768f

**Comment by matkoniecz** [comment]
It also causes notes to be created as anonymous user despite being logged in: https://www.openstreetmap.org/note/2639681 vs https://www.openstreetmap.org/note/2639682

**Comment by tomhughes** [comment]
Yes but the other one was raised earlier, just in the wrong place and I didn't realise until I wondered why a new ticket had been entered s I moved it over here which gave it a new higher number.
